packages/astro/src/vite-plugin-scripts/page-ssr.ts

```diff
@@ -12,7 +12,7 @@ export default function astroScriptsPostPlugin({
 	return {
 		name: 'astro:scripts:page-ssr',
 		enforce: 'post',
-		transform(this, code, id, options) {
+		async transform(this, code, id, options) {
```

Comment:
Nit, but I guess this is no longer doing anything `async`?
```suggestion
		transform(this, code, id, options) {
```

Reply:
Good catch!